app/views/components/docs/document-list.yml

```diff
@@ -0,0 +1,43 @@
+name: Document list
+description: A list of links to documents including document type and when updated.
+body: |
+  The component expects data in a specific format from a content item.
```

Comment:
This is a bit vague - are you referring to the date format here? If so, that probably needs to be made clearer.